alvr/session/src/settings.rs

```diff
@@ -556,23 +556,63 @@ pub enum H264Profile {
     Baseline = 2,
 }
 
-#[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq)]
+#[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq, Debug)]
+pub struct ChromaKeyConfig {
+    #[schema(strings(display_name = "Hue"), suffix = "°")]
+    #[schema(flag = "real-time")]
+    #[schema(gui(slider(min = 0.0, max = 360.0, step = 1.0)))]
+    pub hue_deg: f32,
+
+    #[schema(strings(display_name = "Hue range"), suffix = "°")]
+    #[schema(flag = "real-time")]
+    #[schema(gui(slider(min = 0.0, max = 360.0, step = 1.0)))]
+    pub hue_range_deg: f32,
```

Comment:
tbh this being in degrees doesn't make a ton of sense? I can see the logic of weighting each of the components in a distance function mayyybe, but the default should be 1.0 weight and should be conveyed as a 0~2 weighting, everything made way more sense when I set this value to 180deg